src/App.vue

```diff
@@ -670,11 +670,10 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
       } else {
         const error = new Error('Missing Payment Token')
         this.$root.$emit('save-error-event', error)
-        this.setIsFilingPaying(false)
       }
+      this.setIsFilingPaying(false)
```

Comment:
I moved this into the code block where it is set, and I deleted a duplicate line of code.